### DISCUSSION TITLE
Regarding problems caused by conflicts between software and screenshot software

### DISCUSSION THREAD

**Discussion by wwxiaoqi** [WindowTop/WindowTop-App#382] [Q&A]
Hi, WindowTop.

I really like WindowTop, and I can’t live without it. However, I need to take a lot of screenshots during my work. I’ve noticed that when WindowTop is active, it leaves a maximized, full-screen transparent, and always-on-top interface on the window. This prevents screenshot tools from using UI detection features. I’m curious about the design purpose of this and if there’s a way to solve this issue.

I’ve tried using well-known screenshot tools like [Snipaste](https://www.snipaste.com/download.html) and [Pixpin](https://pixpinapp.com/), but the result is the same.

----

 If the window is pinned to the top, the screenshot tool will return to normal.

**Comment by gileli121** [comment]
Hello, I can't reproduce it. 
I tried to reproduce with [Snipaste](https://www.snipaste.com/download.html) 
Can you record a video that shows this issue?

**Comment by wwxiaoqi** [comment]
https://github.com/user-attachments/assets/38f11a3a-7914-42d9-a8d9-960e7f4a2a25

-----
In the following days, I continued to try many methods and finally found the cause.

Unlike other places, I realized that the reason I was having this problem was because WindowTop prompted me that I did not have sufficient permissions. So, I granted WindowTop the necessary permissions, and the problem arose.

In the video, there are two instances of lag, which is because my recording software is unable to capture the Windows standard user UAC behavior. The two instances of lag are as follows: 

First time: When I used Windows + R to input cmd and then used ctrl + shift + enter to open the administrator. 

Second time: When a prompt at the bottom right corner indicated that WindowTop required administrator permissions.

**Comment by wwxiaoqi** [comment]
> Second time: When a prompt at the bottom right corner indicated that WindowTop required administrator permissions.

~~It is worth mentioning that this issue can occur even without allowing administrative privileges.~~

When a prompt in the bottom right corner indicates that WindowTop requires administrative privileges, just ignore it. After a short wait, this issue will appear.

**Reply by wwxiaoqi** [reply]
@gileli121.

Do you have any good methods and suggestions for removing this window?

**Reply by gileli121** [reply]
@wwxiaoqi 
Thank you for the additional details. It seems I now have much more information. I will attempt to reproduce it soon using this new information.
